torchci/rockset/utilization/__sql/runner_utilization.sql

```diff
@@ -0,0 +1,24 @@
+SELECT
+      FORMAT_ISO8601(
+        DATE_TRUNC(
+            :granularity,
+            started_at AT TIME ZONE :timezone
+        )
+    ) AS started_date,
+  label,
+  SUM(DATE_DIFF('SECOND',started_at,completed_at)) AS duration,
+FROM (SELECT
+  PARSE_TIMESTAMP_ISO8601(started_at) as started_at,
+  PARSE_TIMESTAMP_ISO8601(completed_at) as completed_at,
+  ELEMENT_AT(labels, 1) AS label
+  FROM commons.workflow_job
+  WHERE
+      status = 'completed' AND
+      runner_group_name = 'GitHub Actions'
+  ) AS gha_jobs
+WHERE
+    started_at >= PARSE_DATETIME_ISO8601(:startTime) AND
```

Comment:
I think these queries could run faster if you move `started_at >= PARSE_DATETIME_ISO8601(:startTime)` and `started_at < PARSE_DATETIME_ISO8601(:stopTime)` into the FROM SELECT.  For example,

```
WHERE
    status = 'completed' AND
    runner_group_name = 'GitHub Actions' AND
    started_at >= PARSE_DATETIME_ISO8601(:startTime) AND
    started_at < PARSE_DATETIME_ISO8601(:stopTime)
```

Reply:
Looks like the case, though I have to call `PARSE_DATETIME_ISO8601` explicitly for the both sides of the equation